docs/src/pages.js

```diff
@@ -113,6 +113,7 @@ const pages = [
         subheader: '/components/lab',
         children: [
           { pathname: '/components/about-the-lab' },
+          { pathname: '/components/alerts' },
```

Comment:
Singular, house policy

```suggestion
          { pathname: '/components/alert' },
```

Reply:
ahh.  I wasn't sure where to use singular and where to use plural but now that you point it out it makes sense!